content/articles/how-to-create-a-machine-learning-app-using-turicreate/index.md

```diff
@@ -0,0 +1,413 @@
+ 'muscle stiffness',
+ 'Alopecia',
+ 'Obesity',
+ 'class']
+```
+
+Our target will be the class column.
+
+### Getting Features
+
+```python
+feature_names = ['Age',
+ 'Gender',
+ 'Polyuria',
+ 'Polydipsia',
+ 'sudden weight loss',
+ 'weakness',
+ 'Polyphagia',
+ 'Genital thrush',
+ 'visual blurring',
+ 'Itching',
+ 'Irritability',
+ 'delayed healing',
+ 'partial paresis',
+ 'muscle stiffness',
+ 'Alopecia',
+ 'Obesity']
+```
+
+### Building Machine Learning Model
+
+In this phase, we will start building our model using the given dataset above.
+Before we begin, we need to split our dataset into a training set and a testing set. The training set will be 75%, and the testing set will be 25%.
+
+### Dataset splitting
+
+```python
+train_data,test_data = df.random_split(0.75)
+```
+
+- Shape of the original data.
+
+```python
+df.shape()
+```
+
+Output:
+
+```bash
+(520, 17)
+```
+
+- Shape of our training set.
+
+```python
+train_data.shape
+```
+
+Output:
+
+```bash
+(390, 17)
+```
+
+After splitting our data into a training and a testing set, we can begin building our model.
+
+### Modelling Algorithm
+
+TuriCreate supports different classification algorithms, in our case, we will use the [Logistic regression](https://en.wikipedia.org/wiki/Logistic_regression)
+
+- To use the logistic regression algorithm use the foloowing command:
+
+```python
+logistic_model = tc.logistic_classifier.create(train_data,target='class',features=feature_names)
+```
+
+Output:
+
+```bash
+PROGRESS: Creating validation set data from 75% of train_data.
+
+
+Logistic regression:
+--------------------------------------------------------
+Number of examples: 350
+Number of classes: 2
+Number of feature columns: 16
+Number of unpacked features: 16
+Number of coefficients: 17
+Starting Newton Method
+--------------------------------------------------------
++-----------+----------+--------------+-------------------+---------------------+
+| Iteration | Passes   | Elapsed Time | Training Accuracy | Validation Accuracy |
++-----------+----------+--------------+-------------------+---------------------+
+| 1         | 2        | 0.014390    | 0.927183         | 0.897374            |
+| 2         | 3        | 0.016456    | 0.931734         | 0.843725            |
+| 3         | 4        | 0.018732    | 0.949024         | 0.843725            |
+| 4         | 5        | 0.020935    | 0.957320         | 0.843725            |
+| 5         | 6        | 0.023619    | 0.960135         | 0.843725            |
+| 7         | 8        | 0.026907    | 0.969716         | 0.843725            |
++-----------+----------+--------------+-------------------+---------------------+
+SUCCESS: Optimal solution found.
+```
+
+TuriCreate will be able to use both the training accuracy and Validation accuracy through the iteration. Here we iterate 7 times, and the Training accuracy and Validation accuracy after 7 iterations will be 0.965517 0.842105, respectively.
+
+### Getting the model summary
+
```

Comment:
Give an outline of what to expect from each section/ sub-section. If we jump into the code directly, we lose the interest of the reader.